assets/src/edit-story/components/tooltip/deprecatedTooltip.js

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
This whole file was renamed to `deprecatedTooltip.js` from `index.js`. Need to keep it because the `WithTooltip` is still being used in the editor.

Reply:
Is there a ticket for removing that from the editor?

Reply:
yes, there's a ticket to remove all deprecated code for the editor - think Miina's working on it soon.